applications/nrf_desktop/src/events/hid_event.c

```diff
@@ -32,12 +33,12 @@ static int log_hid_report_event(const struct event_header *eh, char *buf,
 
 			pos += tmp;
 
-			if (pos >= buf_len)
+			if (pos >= HID_EVENT_LOG_BUF_LEN)
 				break;
 		}
 	}
-
-	return pos;
+	EVENT_MANAGER_LOG(eh, "%s", log_strdup (log_buf));
```

Comment:
Also add error handling

Reply:
And remove empty space

Reply:
No removed.

Reply:
as mentioned - there is still excessive space after `log_strdup`

```
log_strdup (log_buf))
          ^
          |
         here
```